docs/how-to/contributing/running_tests.rst

```diff
@@ -58,3 +58,21 @@ You can also run the doctests on any given file. For example, to run the
 doctests for the :code:`docs/tutorials/getting_started/match.rst` file::
 
     $ python -m doctest docs/tutorials/getting_started/match.rst
+
+Performance testing
+-------------------
+
+Performance is not automatically tested as of yet.  To check the performance
+of a change, first install pytest-benchmark::
+
+    $ pip install pytest-benchmark
```

Comment:
```suggestion
    $ python -m pip install pytest-benchmark
```

(I'm sure we're not consistent yet here throughout the docs but nice to catch them when we can :)).